consensus/hotstuff/pacemaker/pacemaker.go

```diff
@@ -88,11 +88,19 @@ func (p *NitroPaceMaker) UpdateCurViewWithQC(qc *flow.QuorumCertificate) (*model
 	return p.gotoView(newView), true
 }
 
+func (p *NitroPaceMaker) ProcessTC(tc *flow.TimeoutCertificate) (*model.NewViewEvent, bool) {
+	panic("not yet implemented")
+}
+
+func (p *NitroPaceMaker) OnPartialTC(curView uint64) {
+	panic("not yet implemented")
+}
+
 // UpdateCurViewWithBlock indicates the pacermaker that the block for the current view has received.
 // and isLeaderForNextView indicates whether or not this replica is the primary for the NEXT view.
 func (p *NitroPaceMaker) UpdateCurViewWithBlock(block *model.Block, isLeaderForNextView bool) (*model.NewViewEvent, bool) {
```

Comment:
This function is no longer required, since the block does not trigger a view change anymore.  Could we just remove this function entirely?